summer_school_private/smb_confusor/package.xml

```diff
@@ -19,5 +19,7 @@
   <depend>sensor_msgs</depend>
   <depend>tf</depend>
   <depend>confusion</depend>
+  <depend>Ceres</depend>
```

Comment:
please remove this, this dependency should not be managed by catkin as some people have ceres already on their system